<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>know_words</class>
 <widget class="QMainWindow" name="know_words">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>know_words</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="dictionary_status_label">
         <property name="text">
          <string>No Dictionary</string>
         </property>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="font_size_label">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="font_size_spinBox">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_dictionary_button">
            <property name="text">
             <string>Add dictionary</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="about_dictionary_button">
            <property name="text">
             <string>About dictionary</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="input_label">
          <property name="text">
           <string>INPUT:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="input_text_edit"/>
        </item>
        <item>
         <widget class="QLabel" name="output_label">
          <property name="text">
           <string>OUTPUT:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="output_text_edit"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="search_button">
          <property name="text">
           <string>SEARCH</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sort_button">
          <property name="text">
           <string>SORT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="random_button">
          <property name="text">
           <string>RANDOM</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
